source/Body.h

```diff
@@ -107,6 +111,8 @@ class Body {
 	float scale = 1.f;
 
 	double alpha = 1.;
+	double distanceVisible = 0.;
+	double distanceInvisible = -1.;
```

Comment:
```suggestion
	double distanceInvisible = 0.;
```
you didnt update it here to go with the check right?

Reply:
yes